src/sdk/config_fs.js

```diff
@@ -839,6 +849,20 @@ class ConfigFS {
         return { parsed_account_data, string_account_data };
     }
 
+    /**
+     * remove_encrypted_secret_key will remove the encrypted_secret_key property from an identity
+     * since we allow to have more than one access key, we hold the index (but most cases are with 1)
+     * @param {object} config_data
+     */
+    remove_encrypted_secret_key(config_data) {
+        let index = 0;
+        for (const access_key of config_data.access_keys) {
+            config_data.access_keys[index] = _.omit(access_key, ['encrypted_secret_key']);
+            index += index;
+        }
+        return config_data;
```

Comment:
it's not really clear what is going on here with the index, especially the line `index += index;`
why not use a regular for (const i=0; ....) instead

Reply:
Changed, thanks.
I thought it might be confusing with the standard loop.